dist/aframe-fence-component.js

```diff
@@ -42,7 +42,7 @@
 /************************************************************************/
 /******/ ([
 /* 0 */
-/***/ function(module, exports) {
+/***/ (function(module, exports) {
```

Comment:
just curious, why do you wrap the function in ()?

Reply:
to be honest with you it was most probably my IDE deciding it was a good idea and me making the commit without realizing :) I can remove it.

Reply:
no worries. looks good to me